nautobot/dcim/models/device_components.py

```diff
@@ -376,6 +376,7 @@ def get_power_draw(self):
             "maximum": self.maximum_draw or 0,
             "outlet_count": PowerOutlet.objects.filter(power_port=self).count(),
             "legs": [],
+            "utilization_data": UtilizationData(numerator=self.allocated_draw or 0, denominator=self.maximum_draw or 0),
```

Comment:
Comparing against https://github.com/netbox-community/netbox/blob/develop/netbox/templates/dcim/rack.html#L221, it looks like the original version of this utilization graph was using the `available_power` of the connected `PowerFeed` as the denominator - so perhaps:

```python
denominator = self.connected_endpoint.available_power or 0
```

?